### ISSUE TITLE
Thermodynamics calculator

### ISSUE THREAD

**Issue by Amit366** [Amit366/FunwithPhysics#98]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Add this  calculator in the thermodynamics section

**Comment by Amit366** [comment]
@souravseal99 first of all it was crashing the website so had to revert it and secondly u need to add calculator for carnot engine, efficiency ,work done and so on.

**Comment by souravseal99** [comment]
> @souravseal99 first of all it was crashing the website so had to revert it and secondly u need to add calculator for carnot engine, efficiency ,work done and so on. 

Yes, initially it was.. but then I made a second PR which resolved all the merge conflicts..

And to your second doubt, you can check in the issue, it's not anywhere mentioned that It's related to the implementation of Karno's Engine, efficiency and all.

**Comment by souravseal99** [comment]
It would be great, if you could be a little bit more specific about the issues from next time...

**Comment by Amit366** [comment]
@souravseal99 the second PR too had the KTG calculator missing that's why it was reverted and the same I commented in the PR and also on discord. 

And the second one is carnot engine and efficiency all this come under thermodynamics topic and it is understood by the meaning of thermodynamics calculator
